lib/scrabble.rb

```diff
@@ -9,5 +10,63 @@ def point_values
       "U"=>1, "V"=>4, "W"=>4, "X"=>8,
       "Y"=>4, "Z"=>10
     }
+    @multi_score
```

Comment:
are these three instance variables things that could be calculated on the fly, or remain just return values of methods?